microsoft-365/compliance/create-a-custom-sensitive-information-type.md

```diff
@@ -180,6 +180,8 @@ For a scanned item to satisfy rule criteria, the number of unique instances of a
 
 For example, if you want the rule to trigger a match when at least 500 unique instances of a SIT are found in a single item, set the **min** value to `500` and the **max** value to `Any`.
 
+Note: for EDM, we support up to 100 EDM SIT evaluations, thus policies should not be written with a Min or Max value greater than 100 for EDM SITs.
```

Comment:
```suggestion
> [!NOTE]
> We support up to 100 exact data match (EDM) evaluations, thus policies should not be written with a **min** or **max** value greater than 100 for EDM SITs.
```

Reply:
Overall looks fine but let's please add "instance count" - i.e., ...written with a **min** or **max** instance count value greater than 100 for EDM SITs

Reply:
I changed the suggested change accordingly. Could you please commit? Thanks.